### PR TITLE
Add statistics on total active countries, users, projects to DimagiSphere

### PR DESCRIPTION
**Pull request by rickypchen**
@czue @NoahCarnahan 

Users can see in the legend statistics on total number of countries CommCare is deployed in, total number of projects, and total number of active mobile users

screen shot 2016-08-17 at 5 22 20 pm

### CODE REVIEW

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 1]
```diff
@@ -35,12 +37,14 @@ var projectMapInit = function(mapboxAccessToken) {
...
                         maxNumProjects = projects_per_country[country];
                     }
+                    totalNumProjects += projects_per_country[country];
                 });
 
                 Object.keys(users_per_country).map(function(country) {
                     if (users_per_country[country] > maxNumUsers) {
                         maxNumUsers = users_per_country[country];
                     }
+                    totalNumUsers += users_per_country[country]
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 2]
```diff
@@ -277,12 +293,26 @@ var projectMapInit = function(mapboxAccessToken) {
 
     legend.addTo(map);
 
+
+    var stats = L.control({position: 'bottomright'});
+
+    stats.onAdd = function (map) {
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: 'map' is defined but never used (no-unused-vars)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 3]
```diff
@@ -277,12 +293,26 @@ var projectMapInit = function(mapboxAccessToken) {
...
+        var div = L.DomUtil.create('div', 'info legend');
+        div.innerHTML += '<p><b>Statistics</b></p>'
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 4]
```diff
@@ -277,12 +293,26 @@ var projectMapInit = function(mapboxAccessToken) {
...
 
+
+    var stats = L.control({position: 'bottomright'});
+
+    stats.onAdd = function (map) {
+        var div = L.DomUtil.create('div', 'info legend');
+        div.innerHTML += '<p><b>Statistics</b></p>';
+        div.innerHTML += '<p>Number of Active Countries: ' + dataController.getNumActiveCountries() +  '</p>';
+        div.innerHTML += '<p>Number of Active Mobile Users: ' + dataController.getNumUsers() +  '</p>';
+        div.innerHTML += '<p>Number of Active Projects: ' + dataController.getNumProjects() +  '</p>';
```
Comment (czue) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: eventually we should probably translate these

**Comment by czue** [comment]
looking good! kicked build.

**Comment by NoahCarnahan** [comment]
Looks good. Travis is failing on all builds at the moment, but once we've straightened that out we can restart the build and merge when it passes.

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js (hunk 2 above)
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: 'map' is defined but never used (no-unused-vars)

**Comment by snopoke** [comment]
merged master to fix build

**Pull request merged**